### DISCUSSION TITLE
Loading .txt file for stimulus presentation

### DISCUSSION THREAD

**Discussion by awb99cu** [jspsych/jsPsych#1417] [Q&A] [answered]
I'm new to JSPsych, but I've been trying to construct a free recall experiment in which word lists are loaded into javascript from a text file on the host computer. I've tried a number of solutions such as fileReader and ajax, but I can't get any of them to work in Chrome. 

Is there a way to read text files into jsPsych as one would an image file, and present them serially as a list?

Thank you,
Adam

**Comment by jodeleeuw** [comment]
Converting this to a discussion

**Comment by jodeleeuw** [comment, marked as answer]
Hi Adam,

I recorded [a video tutorial dealing with loading text files into jsPsych](https://www.youtube.com/watch?v=T3GCZWarEsY). I'd start there and follow up here with any questions.

**Comment by awb99cu** [comment]
Thank you Josh! I was able to address the issue using the information
provided in your video.

Best,
Adam

On Wed, Jan 27, 2021 at 11:25 AM Josh de Leeuw <notifications@github.com>
wrote:

> Hi Adam,
>
> I recorded a video tutorial dealing with loading text files into jsPsych
> <https://www.youtube.com/watch?v=T3GCZWarEsY>. I'd start there and follow
> up here with any questions.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jspsych/jsPsych/discussions/1417#discussioncomment-315206>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASRTVKRS54LQNIPIZVYPQS3S4A47XANCNFSM4WVRL3QA>
> .
>


-- 
Ph.D. Candidate
Psychology & Cognitive Science
Cornell University | Uris Hall B107
